packages/dependency-royalty-management/contracts/MultiRoyaltyDistributor.sol

```diff
@@ -105,14 +114,14 @@ abstract contract MultiRoyaltyDistributor is IEIP2981, IMultiRoyaltyDistributor,
     /// @notice returns the royalty recipients for each tokenId.
     /// @dev returns the default address for tokens with no recipients.
     /// @param tokenId is the token id for which the recipient should be returned.
-    /// @return addresses of royalty recipient of the token.
-    function getRecipients(uint256 tokenId) public view returns (Recipient[] memory) {
+    /// @return defaultRecipient addresses of royalty recipient of the token.
+    function getRecipients(uint256 tokenId) public view returns (Recipient[] memory defaultRecipient) {
         address payable splitterAddress = _tokenRoyaltiesSplitter[tokenId];
         (address payable _defaultRoyaltyReceiver, ) = IRoyaltyManager(royaltyManager).getRoyaltyInfo();
         if (splitterAddress != address(0)) {
             return IRoyaltySplitter(splitterAddress).getRecipients();
         }
-        Recipient[] memory defaultRecipient = new Recipient[](1);
+        defaultRecipient = new Recipient[](1);
         defaultRecipient[0] = Recipient({recipient: _defaultRoyaltyReceiver, bps: TOTAL_BASIS_POINTS});
         return defaultRecipient;
```

Comment:
The function does not always return `defaultRecipient` when the splitter is non-zero the return value is different.
I would suggest renaming the variable to `recipients`